tests/subsys/sd/sdio/boards/arduino_giga_r1_stm32h747xx_m7.overlay

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (c) 2025 EXALT Technologies.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,console = &cdc_acm_uart0;
+	};
+
+	aliases {
+		sdhc0 = &sdhc;
+	};
+};
+
+&usbotg_fs {
+	status = "okay";
+	pinctrl-0 = <&usb_otg_fs_dm_pa11 &usb_otg_fs_dp_pa12>;
+	pinctrl-names = "default";
+
+	cdc_acm_uart0: cdc_acm_uart0 {
+		status = "okay";
+		compatible = "zephyr,cdc-acm-uart";
+	};
+};
+
+&cdc_acm_uart0 {
+	status = "okay";
+};
```

Comment:
I'd be surprised it is required to put this in all samples using this board.
@pillo79 Can you advise on how this should be handled ?